From 7ef6975f50de5430bca938ad51d00f19e51a2be1 Mon Sep 17 00:00:00 2001
From: Athanasios Oikonomou <athoik@gmail.com>
Date: Thu, 4 Jul 2013 20:12:04 +0300
Subject: [PATCH 1/1] Set only by hardware supported audio mpeg 4 profile

Only mpeg 4 lc profile is supported by the hardware. Other profiles
will be decoded by software. Based on betacentauri patches.
---
 src/gstdvbaudiosink.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/gstdvbaudiosink.c b/src/gstdvbaudiosink.c
index 93cb587..2b5fe73 100644
--- a/src/gstdvbaudiosink.c
+++ b/src/gstdvbaudiosink.c
@@ -196,7 +196,7 @@ GST_STATIC_PAD_TEMPLATE (
 		X_RAW_INT(24,24)
 		X_RAW_INT(32,24)
 		X_RAW_INT(32,32)
-		"audio/mpeg; "
+		"audio/mpeg, profile = (string) lc; "
 		"audio/x-ac3; "
 		"audio/x-private1-ac3; "
 		"audio/x-dts; "
-- 
1.7.9.5

